Jenkinsfile

```diff
@@ -3,10 +3,10 @@
 buildPlugin(useContainerAgent: true,
         configurations: [
                 [platform: 'linux', jdk: '11'],
-                [platform: 'windows', jdk: '8'],
+                [platform: 'windows', jdk: '11'],
```

Comment:
Don't need to also run on Window Java 11.  One platform for Java 11 is enough.
```suggestion
```

Reply:
> Don't need to also run on Window Java 11. One platform for Java 11 is enough.

Hey @MarkEWaite just committed the recommended changes. waiting for you to reply the CIi job